include/qemu/log_instr.h

```diff
@@ -67,6 +85,10 @@ struct cpu_log_instr_info;
 typedef struct {
     /* CPU tracing user mode only enable flag */
     bool user_mode_tracing;
+    /* /\* Are we switching mode? *\/ */
```

Comment:
Are you planning to uncomment these in a follow up commit?

Reply:
I meant to delete them but forgot about it, thanks for poiting it out.